cotk/metric/ngram_perplexity.py

```diff
@@ -13,14 +15,20 @@ class NgramFwBwPerplexityMetric(MetricBase):
 			are passed to :func:`forward` by ``dataloader.data["test"][self.reference_test_key]``.
 		{MetricBase.GEN_KEY_ARGUMENTS}
 	'''
-	def __init__(self, dataloader, ngram, reference_test_key, gen_key="gen"):
+	def __init__(self, dataloader, ngram, reference_test_key, gen_key="gen", cpu_count=None):
 		super().__init__()
 		self.dataloader = dataloader
 		self.ngram = ngram
 		self.reference_test_key = reference_test_key
 		self.gen_key = gen_key
 		self.hyps = []
 		self.refs = []
+		if cpu_count is not None:
+			self.cpu_count = cpu_count
+		elif "CPU_COUNT" in os.environ and os.environ["CPU_COUNT"] is not None:
+			self.cpu_count = os.environ["CPU_COUNT"]

```

Comment:
self.cpu_count = int(os.environ["CPU_COUNT"])

Reply:
remove these lines if not necessary, because it is passed to KnerserNeyInterpolated